static/templates/settings/deactivation_user_modal.hbs

```diff
@@ -9,6 +9,6 @@
     </div>
     <div class="modal-footer">
         <button class="button rounded" data-dismiss="modal">{{t "Cancel" }}</button>
-        <button class="button rounded btn-danger do_deactivate_button">{{t "Deactivate now" }}</button>
+        <button class="button rounded btn-danger do_deactivate_button">{{t "Confirm" }}</button>
```

Comment:
<img width="427" alt="deactivate_other_user_confirm_modal" src="https://user-images.githubusercontent.com/59444243/113510116-ca70c600-9576-11eb-909b-9dfe64c0d98b.PNG">

Reply:
Well, it's not a regression, so I went ahead and merged, but that message isn't great.  The main thing that's not good is the fake emails here.

Reply:
I'm a bit confused -- @timabbott what do you mean by "fake emails"?

Ideally, I think what I'd want to see is something like this:

**Header row**: Deactivate user user's-name <user's-email>
**Body**:
"Deactivating user user's-name <user's-email> will immediately log them out. Their password will be cleared from our systems, and any bots they maintain will be disabled.

Are you sure you want to deactivate this user?"

Reply:
@timabbott Does "fake emails" refers to "user6@zulipdev.com"? (Maybe real email is "AARON@zulip.com" that we show in `admin_users_table` email column).

Reply:
Yes, exactly.  We should be using `.delivery_email`, not `.email`, for this notice.